rules/invite-collaborator.ts

```diff
@@ -5,7 +5,7 @@ Holy buckets, @${username} — we just merged your PR to Gatsby! 💪💜
 
 Gatsby is built by awesome people like you. Let us say “thanks” in two ways:
 
- 1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (Currently we’ve got a couple t-shirts available, plus some socks that are really razzing our berries right now.)
+ 1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of Tier 1 swag. Pick from a selection of custom Gatsby t-shirts, stickers, hats, and much more. After 5 merged PRs you will unlock one free piece of Tier 2 swag which include epic items like hoodies, pajama pants, and beverage mugs!
```

Comment:
We dropped the use of "tier" as a description of the swag, so this edit cleans that up. I also simplified the language around level 2 swag and added a link for more info.

```suggestion
 1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. We’ve got Gatsby t-shirts, stickers, hats, scrunchies, and much more. (You can also unlock _even more_ free swag with 5 contributions — wink wink nudge nudge.) See [gatsby.dev/swag](https://gatsby.dev/swag) for details.
```

Reply:
For sure! I dig it!